adapters/repos/db/vector/hnsw/commit_logger_noop.go

```diff
@@ -63,6 +63,8 @@ func (n *NoopCommitLogger) Drop() error {
 	return nil
 }
 
+func (n *NoopCommitLogger) Shutdown() {}
```

Comment:
shouldn't `hnswCommitLogger` implement `Shutdown()` method as well?

Reply:
Hey, good question! It [already does](https://github.com/semi-technologies/weaviate/blob/8054fbb07d90ecb04ed229403e7e93e96cb460b0/adapters/repos/db/vector/hnsw/commit_logger.go#L388) :)